pkg/rancher-desktop/utils/protocols.ts

```diff
@@ -5,6 +5,10 @@ import { app, ProtocolRequest, ProtocolResponse, protocol } from 'electron';
 
 import { isDevEnv } from '@pkg/utils/environment';
 import Latch from '@pkg/utils/latch';
+import Logging from '@pkg/utils/logging';
+import paths from '@pkg/utils/paths';
+
+const console = Logging.background;
```

Comment:
Until we solve the problem with null bytes overwriting the first part of `background.log` I'd prefer writing to a different log file.